Makefile

```diff
@@ -165,6 +167,38 @@ build-deepspeed-gpu:
 		-t $(NGC_REGISTRY)/$(GPU_DEEPSPEED_ENVIRONMENT_NAME)-$(VERSION) \
 		.
 
+# This builds deepspeed environment off of a patched version of EleutherAI's fork of DeepSpeed 
+# that we need for gpt-neox support.
+.PHONY: build-deeperspeed-gpu
```

Comment:
I'm not so sure about the name deeperspeed and it's clarity. Would it be wrong to use EleutherAI's name here?